doc/userguide/partials/counters.rst

```diff
@@ -0,0 +1,95 @@
+flow.mgr.full_hash_pass
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Number of times the flow hash table has been fully scanned for timed out flows.
+
+The flow manager normally scans it in small steps (slices).
+
+flow.mgr.rows_maxlen
+~~~~~~~~~~~~~~~~~~~~
+
+The flow hash table is set up as an array of lists. There is a linked list at
+each bucket. The length of this list affects lookup performance, so it should
+be short. In the ideal case it will have a length of 1. If this number gets
+high, say > 30, its possible the hash table size needs to be increased.
+
+flow.mgr.flows_checked
+~~~~~~~~~~~~~~~~~~~~~~
+
+Number of flows checked by the flow manager to see if they are timed out.
+
+flow.mgr.flows_notimeout
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Number of flows that were checked, but did not reach their timeout time yet.
+
+flow.mgr.flows_timeout
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Number of flows that were checked, and did their timeout time.
+
+flow.mgr.flows_evicted
+~~~~~~~~~~~~~~~~~~~~~~
+
+Number of flows evicted from the flow hash by the flow manager.
```

Comment:
Is there any discussion on how to interpret these values? Are cumulative since Suricata started? Or since the last stat was printed?

Reply:
All but the `avg` counter are cumulative. Would be good to somehow annotate them in a standard way